assets/src/domain/shared/predicates/tickets/filterTicketEntities/validInfiniteQuantity.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { differenceInMinutes } from 'date-fns';
+
+/**
+ * @param {Object} ticketEntity    event ticket object
+ * @return {boolean} true if qty property is valid and unlimited
+ */
+const validInfiniteQuantity = (ticketEntity) => {
+	return (
+		isModelEntityOfModel(ticketEntity, 'ticket') && (ticketEntity.qty === 'INF' || ticketEntity.qty === Infinity)
```

Comment:
TS will remove the need to be doing this all over the place:  `isModelEntityOfModel(ticketEntity, 'ticket')`